compiler/rustc_serialize/tests/leb128.rs

```diff
@@ -25,13 +26,15 @@ macro_rules! impl_test_unsigned_leb128 {
                 let n = $write_fn_name(&mut buf, x);
                 stream.extend(&buf[..n]);
             }
+            let stream_end = stream.len();
+            stream.extend(b"rust-end-file");
```

Comment:
```suggestion
            stream.extend(FOOTER);
```

with `FOOTER` being made crate-pub?

Reply:
```
error[E0603]: constant `FOOTER` is private
  --> compiler/rustc_serialize/tests/leb128.rs:3:30
   |
3  | use rustc_serialize::opaque::FOOTER;
   |                              ^^^^^^ private constant
   |
note: the constant `FOOTER` is defined here
  --> /home/ben/rust/compiler/rustc_serialize/src/opaque.rs:20:1
   |
20 | pub(crate) const FOOTER: &[u8] = b"rust-end-file";
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Integration tests are a separate crate :upside_down_face: 
I'll make it fully `pub`.